examples/src/main/java/io/grpc/examples/helloworld/HelloWorldServer.java

```diff
@@ -89,7 +89,7 @@ public static void main(String[] args) throws IOException, InterruptedException
     server.blockUntilShutdown();
   }
 
-  private class GreeterImpl extends GreeterGrpc.GreeterImplBase {
+  static class GreeterImpl extends GreeterGrpc.GreeterImplBase {
```

Comment:
I know that it references, but *why* does it need access to it?  If you reference the implementation from here, then you basically bypass all the server setup and other code in this file.  In effect, you aren't really testing this file, just an inner class.